### DISCUSSION TITLE
Any reason to keep /watchtower on a different network compared to the other containers?

### DISCUSSION THREAD

**Discussion by wiffleman** [containrrr/watchtower#1222] [Q&A] [answered]
My use case: all containers essentially in one docker-compose stack, following smarthomebeginner guides to route behind Traefik and Cloudflare. 

In his builds, watchtower is placed under a separate network within Docker. Ie. other containers: network: t2_proxy; whereas watchtower is network: default.

I have added watchtower successfully under the t2_proxy network and all seems to be working well but am just curious as to why smarthomebeginner would have placed it under a diff network to begin with?

**Comment by simonaronsson** [comment, marked as answer]
I don't know about their rationale. I'm assuming as watchtower does not (by default) receive any incoming connections, there really isn't any need to run it in an isolated network. Should be fine to place it in whichever, or both, as long as it has egress internet access.

**Reply by piksel** [reply]
Yeah, since it's just the `default` network, I think it's more a case of being explicit for the containers that need to be in the same network, and then just putting watchtower "wherever".
